Makefile

```diff
@@ -321,3 +322,15 @@ bump-minor-version:
 	  git commit $(VERSION_FILE) \
 	    -m "Bump minor version to $$(cat $(VERSION_FILE))" \
 	    -m 'Commit generated with `make bump-minor-version`'
+
+# Copy the ECP file use for releases then patch in some tweaks
+# (Requires access to gitlab.cee.redhat.com and a token in GITLAB_TOKEN)
+END_POINT=https://gitlab.cee.redhat.com/api/v4/projects/78877/repository/files/
+ECP_PATH=config%2Fstone-prd-rh01.pg1f.p1%2Fproduct%2FEnterpriseContractPolicy%2Fregistry-rhtap-contract.yaml
+ECP=curl -s --header "PRIVATE-TOKEN: $(GITLAB_TOKEN)" "$(END_POINT)$(ECP_PATH)/raw?ref=main"
+sync-custom-ecp:
```

Comment:
Not sure if this belongs in this repo. Also not sure if when we do need to sync, we will remember this make target exists :shrug:

Reply:
Yeah, it's possibly overkill. I added a comment about the make target. :shrug: